cinder/autogenerated_client.go

```diff
@@ -0,0 +1,1821 @@
+package cinder
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type RequestHandlerFn func(*http.Request) (*http.Response, error)
```

Comment:
I'd like to see a bunch of doc describing the auto generation process, a link to the generation utility, where the WADL is sourced from etc.